docs/specs/code/TREZOR_INTEGRATION.md

```diff
@@ -0,0 +1,247 @@
+# Abstract
+
+1. Users would be able to use a Trezor Hardware Wallet with Yoroi Wallet.
+2. Hardware wallet can be integrated by creating new Yoroi wallet, by using `Connect to Trezor Hardware Wallet` on `Add Wallet` page.
+3. [TrezorConnect](https://github.com/trezor/connect) API will be used to communicate with hardware device.
+
+# Motivation
+
+1. Since private key never leave the hardware wallet so it's considered as one of the most secured way of using cryptocurrency wallets.
+2. No need to remember spending passowrd, so its easy to use.
```

Comment:
I would mention that you still need to remember a pincode, so it’s easier but not necessarily easy.

Reply:
@nicarq 
we can add that in drawback block.